managers/opportunity/index.js

```diff
@@ -41,20 +41,139 @@ class opportunityManager {
     }
   }
 
+  // http://localhost:3030/opportunity?limit=4&page=2, Example of how to use the Get Method
+
+  //  page => Means the Current Page that the User Wants
+  //  limit => The count of pages that should be rendered starting from the given page
+
+  //  For the above specified example , we Get 4,5,6,7 (having Opprtunity_id as 3,4,5,6 and respectively) .
+
+  // so in short for limit 4 we get results as page 1 = {0,1,2,3} ,  page 2 = {4,5,6,7} and so on .
+
   async getOpportunities(queryObject) {
     try {
       // Filter based on opportunityType
       if (queryObject.type) {
         queryObject['opportunityType'] = queryObject.type;
         delete queryObject.type;
       }
-      console.log('Values in QueryString', queryObject);
 
-      let fetchedOpportunitiesQuery = this.opportunity.find(queryObject);
-      // fetchedOpportunitiesQuery.select('-__v');
-      // fetchedOpportunitiesQuery.select('-_id');
+      console.log('Values of QueryString', queryObject);
+
+      // Here page means which page to start from and limit specifies the number of documnents to be
+      // rendered starting from this page
+
+      let { limit, page } = queryObject;
+
+      // The case when both page and limit are not specified we assign both of them default values
+      if (page == undefined && limit == undefined) {
+        limit = 1;
+        page = 1;
+      }
+
+      //Only limit is 'undefined or 0' , so it is assigned a default value of 1
+      if (limit == undefined) {
+        limit = 1;
+      }
+
+      // The case when starting page is not mentioned or it is Put in as 0 , we use default value
+      if (page == undefined) {
```

Comment:
In comment, you have written that it covers the case of page = 0  also, but the `if` condition covers only `undefined`